backend/src/main/java/com/ambacam/rest/ApiConstants.java

```diff
@@ -9,4 +9,7 @@
 	public static final String STATUS_REQUETE_COLLECTION = "/status-requetes";
 
 	public static final String STATUS_REQUETE_ITEM = STATUS_REQUETE_COLLECTION + "/{statusRequeteId}";
+
+	public static final String LOG_COLLECTION = "/logs";
```

Comment:
In the swagger the path is: "/log/search"

Reply:
You are right,

but /log/search is an error in the swagger. It should be /logs/search